docs/Customization/Incorporating-HLS.rst

```diff
@@ -0,0 +1,69 @@
+.. _incorporating-hls:
+
+Incorporating HLS
+============================
+
+High Level Synthesis (HLS) is a method for iterating quickly on 
+different hardware algorithms that automatically generates an RTL 
+circuit to match a specification in a high level language like C.
+
+Here, we will integrate an HLS-generated accelerator that computes 
+the Great Common Denominator (GCD) of two integers. This tutorial 
+builds on the sections :ref:`mmio-accelerators` and 
+:ref:`incorporating-verilog-blocks`. The code for this example can 
+be found in ``/generators/hls-example``
+
+Adding an HLS project
+---------------------------------------
+
+In this tutorial, we use Vitis HLS, version 2023.2. 
+
+Our project consists of 3 HLS files:
+* C program of the GCD algorithm: ``accel/HLSAccel.cpp``
+* Header file: ``accel/HLSAccel.hpp``
+* TCL script to run Vitis HLS: ``run_hls.tcl``
+
+To generate the verilog files, as well as synthesis reports, run:
+
+.. code-block:: none
+    vitis_hls run_hls.tcl
+
+The files can be found in a generated folder named proj\_\<your\_project\_name>, 
+in our case, ``proj_gcd_example``.
+
+In our case, we include a ``Makefile`` to script running HLS. To generate the 
+verilog files using the Makefile, run:
```

Comment:
Below you mention this also does the "move files to vsrc" step. So maybe I would add some extra saying "to script running HLS (as well as copying files around as explained later)"